File: PlatformSpecificCode/specific-impl/windowsphone/com/neomades/specific/AbstractGeneratedCompass.cs
using com.neomades.win.ext;

namespace com.neomades.specific
{
	public abstract class AbstractGeneratedCompass : java.lang.Object
	{

		public com.neomades.specific.CompassListener listener;

		public virtual bool isSupported()
		{
			return false;
		}

		public virtual void setCompassListener(com.neomades.specific.CompassListener listener)
		{
			this.listener = listener;
		}

	}
}

File: PlatformSpecificCode/specific-impl/android/com/neomades/specific/AbstractGeneratedCompass.java
package com.neomades.specific;

public abstract class AbstractGeneratedCompass {

	protected com.neomades.specific.CompassListener listener;

	public boolean isSupported() {
		return false;
	}

	public void setCompassListener(com.neomades.specific.CompassListener listener) {
		this.listener = listener;
	}

}